### PR TITLE
Documented test classes

### PR DESCRIPTION
**Pull request by dari1495**
Also added name nia and gradle to README

### CODE REVIEW

**Comment by fjlopez** [comment]
Accepted :+1:

File: src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java [hunk 1]
```diff
@@ -22,7 +22,13 @@
     @Autowired
     private HelloController controller;
 
-
+    /**
+     * This test method checks wether the controller was creted successfully and its contents
```
Comment (MarcosCM) [inline src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: Wether --> Whether

File: src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java [hunk 2]
```diff
@@ -22,7 +22,13 @@
...
+     * are correct.
+     * If the controller is not as expected and does not contain a correct 'message' value the
+     * test will not pass.
+     * @throws Exception
```
Comment (MarcosCM) [inline src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: The case that throws an Exception should be specified

Comment (fjlopez) [reply src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: @dari1495 could you apply these suggestions and do a pull request again?

**Pull request merged**